inc/composer/class-docker-compose-generator.php

```diff
@@ -375,7 +375,7 @@ protected function get_service_kibana() : array {
 	protected function get_service_s3() : array {
 		return [
 			's3' => [
-				'image' => 'minio/minio:RELEASE.2020-03-19T21-49-00Z',
+				'image' => 'minio/minio:RELEASE.2020-12-29T23-29-29Z',
```

Comment:
Guessing this is back compatible?

Reply:
Yeah not sure tbh. Damn these date version numbers